### ISSUE TITLE
Allow project to be adopted by the jupyterhub github organization?

### ISSUE THREAD

**Issue by consideRatio** [scottyhq/jupyter-cloud-scoped-creds#4]
@scottyhq @yuvipanda this project is a jupyter_server / notebook extension that is serving a purpose for jupyterhub in cloud environments (currently only AWS) where cloud provider specific credentials are provided to the user servers, allowing them to work against object storage for example.

I think this could be in scope to maintain within the jupyterhub github organization, and I would be a jupyterhub org member willing to champion an adoption to it help maintain this project long term from within the organization. This project is of direct relevance for hubs deployed by 2i2c.org where I work, but its also relevant for the broader jupyterhub community just like for example https://github.com/jupyterhub/gh-scoped-creds that has also been adopted from Yuvi to the jupyterhub organization.

### Technical roadmap

- Provide sphinx based documentation to cover
  - What the project is about
  - How it works
  - Security aspects
  - Admin user instructions (someone setting this up)
  - End user instructions (someone using this)
  - Changelog
- Add support for GCP credentials
- Add test of at least the extension setup
- Setup CI/CD for autoformatting, tests, and release automation
- Publish docs to readthedocs
- Publish package to PyPI

### Question

@scottyhq @yuvipanda are you okay with letting this be adopted by jupyterhub, and that I would go ahead and ask about if jupyterhub is willing to adopt this project in jupyterhub/team-compass?

**Comment by scottyhq** [comment]
> @scottyhq @yuvipanda are you okay with letting this be adopted by jupyterhub, and that I would go ahead and ask about if jupyterhub is willing to adopt this project in jupyterhub/team-compass?

Absolutely! I don't really have time to work on it beyond this first prototype experiment. Happy to transfer to the 2i2c github org too in the meantime if that is helpful. The roadmap looks great

**Comment by yuvipanda** [comment]
YEP!
